packages/mockData/mockGrossVolumeReport.js

```diff
@@ -0,0 +1,125 @@
+export const mockGrossVolumeReport = {
```

Comment:
I don't think this is supposed to get checked in, in fact I thought it was added to our `.gitignore`. But since that seems to not be the case I think we should remove these changes and add it to the `.gitignore`

Reply:
I removed it from this branch - but yeah I'm not sure why it's not being caught by the `.gitignore` file. 🙃